core/src/main/scala/org/http4s/HttpVersion.scala

```diff
@@ -36,12 +36,20 @@ final case class HttpVersion private[HttpVersion] (major: Int, minor: Int)
 }
 
 object HttpVersion {
-  val `HTTP/1.0` = new HttpVersion(1, 0)
-  val `HTTP/1.1` = new HttpVersion(1, 1)
-  val `HTTP/2.0` = new HttpVersion(2, 0)
+  val HTTP_1_0 = new HttpVersion(1, 0)
+  @deprecated("Rename to HTTP_1_0", "0.22.0-M6")
+  def `HTTP/1.0` = new HttpVersion(1, 0)
```

Comment:
(Heads to the bikeshed)

Proper Scala naming convention would have been `Http1_0`.